i18n/en.pot

```diff
@@ -17,12 +17,47 @@ msgstr "Publish"
 msgid "Unpublish"
 msgstr "Unpublish"
 
+msgid "Pending"
+msgstr "Pending"
+
 msgid "Name is already in use"
 msgstr "Name is already in use"
 
+msgid "Country required"
+msgstr "Country required"
+
+msgid "Name required"
+msgstr "Name required"
+
+msgid "Level required"
+msgstr "Level required"
+
+msgid "Your catchment areas are being created. It should be ready in a few minutes"
+msgstr "Your catchment areas are being created. It should be ready in a few minutes"
+
+msgid "errors were found"
+msgstr "errors were found"
+
+msgid ""
+"Click proceed if you want to continue. The sites with errors will be "
+"removed if you proceed or click cancel to go back."
```

Comment:
I think it would be good to have a comma after `proceed` to clarify that the sites with errors won't be removed if you click cancel to go back.